subsys/bluetooth/controller/bluetooth_controller.cmake

```diff
@@ -0,0 +1,60 @@
+set_target_properties(subsys_bluetooth PROPERTIES EXCLUDE_FROM_ALL False)
```

Comment:
Why do we want to include subsys_bluetooth in all?

Reply:
You're not including it in all, instead you're not excluding it.
but this line should not be here.

and i'm actually a bit puzzled how it got in here so I will remove as well as the line in bluetooth_common.cmake.